.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```

Comment:
I usually open a blank issue, do we really have to remove it entirely?

Reply:
Same

Reply:
"Report a generic issue" is pretty much that, that is why I disabled to avoid possible confusion. But as I don't care enough -- I now committed suggested making it available as well.